custom/lib/User.php

```diff
@@ -115,14 +136,19 @@ public function deactivate(string $userId): PassageUser
      * Update a user.
      *
      * @param string $userId The Passage user ID
-     * @param UpdateUserArgs $args The updated user information
+     * @param UpdateUserArgs $options The updated user information
      * @return PassageUser Passage User Object
+     * @throws InvalidArgumentException Invalid parameter value
      * @throws PassageError
      */
-    public function update(string $userId, UpdateUserArgs $args): PassageUser
+    public function update(string $userId, UpdateUserArgs $options): PassageUser
     {
+        if (!$userId) {
+            throw new InvalidArgumentException('userId is required');
+        }
+
```

Comment:
Exception for `options`

Reply:
do you think adding it would still be necessary given the type hint for a non-nullable method argument that has no default value?